src/components/xl-newsletter/xl-newsletter.jsx

```diff
@@ -10,24 +9,38 @@ import { classnames } from "../../helpers";
 import * as styles from "./xl-newsletter.module.scss";
 
 const XlNewsletter = () => (
-  <div className={styles.container} >
-    <div className={styles.logoContainer}>
-      <Link to="/" className={styles.linkLogoXl}>
-        <StaticImage    
-          src="../../../static/images/newsletter-xl/logo-xl.png"
-          alt="Xmartlabs logo"
-          layout="fixed"
-          width={100}
-          height={100}
-        />
-      </Link>
-    </div>
+    <div className={styles.container} >
+      <div className={styles.logoContainer}>
+        <Link to="/" className={styles.linkLogoXl}>
+          <StaticImage    
+            src="../../../static/images/newsletter-xl/logo-xl.png"
+            alt="Xmartlabs logo"
+            layout="fixed"
+            width={100}
+            height={100}
+          />
+        </Link>
+      </div>
     <div className={styles.informationContainer}>
       <h2 className="text__heading__two__neutral100">Xmartlabs´ Newsletter</h2>
       <p className={classnames("text__paragraph__two__defaultGray", styles.textNewsletter)}>Subscribe to our newsletter and get updates on AI, Computer Vision as well as mobile and web development.</p>
       <div className={styles.subscribeContainer}>
-        <input placeholder="Type your email..." className={classnames(styles.inputNewsletter, "text__placeholder__grayThree")} />
-        <button className={classnames(styles.buttonNewsletter, "text__paragraph__small__neutral100")} onClick={() => {}}>SUBSCRIBE</button>
+          <input 
+            placeholder="Type your email..." 
+            className={classnames(styles.inputNewsletter, "text__placeholder__grayThree")} 
+            onChange={() => {}}
```

Comment:
Why would we need this empty callback?

Reply:
In reality, I was trying something and I changed it. But, it doesn't have any importance.